### PR TITLE
{Fixed Suggested points and created setting and profile page}First milestone01 revision01

### PR DESCRIPTION
**Pull request by kshekh**
Please get pull and check First Revisions

### CODE REVIEW

**Review by joeelia — APPROVED** [review]
Good enough

**Comment by joeelia** [comment]
Got it to build. Deleting old PRs - [first-milestone01-revision01.browtricks-ui-components.pages.dev](https://first-milestone01-revision01.browtricks-ui-components.pages.dev/)

File: .gitignore [hunk 1]
```diff
@@ -27,4 +28,5 @@ coverage
 *.sln
 *.sw?
 
-package-lock.json
\ No newline at end of file
+package-lock.json
```
Comment (giteshk11) [inline .gitignore:31]: Can you delete package-lock.json and use yarn?

File: package.json [hunk 2]
```diff
@@ -1,42 +1,34 @@
...
+    "build": "vite build",
+    "preview": "vite preview",
+    "test:unit": "vitest --environment jsdom --root src/"
+  },
+  "dependencies": {
+    "core-js": "^3.8.3",
+    "pinia": "^2.0.26",
+    "postcss-cli": "^10.1.0",
+    "postcss-nested": "^6.0.0",
+    "postcss-nesting": "^10.2.0",
```
Comment (giteshk11) [inline package.json:16]: Move these to`devDependencies`

File: src/App.vue [hunk 3]
```diff
@@ -1,85 +1,67 @@
...
-    padding: 1rem 0;
-    margin-top: 1rem;
-  }
-}
-</style>
+<template>
+    <router-view></router-view>
+   
+ </template>
+
```
Comment (giteshk11) [inline src/App.vue:5]: Not formatted correctly, use prettier

File: src/assets/global.scss [hunk 4]
```diff
@@ -32,6 +32,10 @@
     @apply bg-black/20;
```
Comment (giteshk11) [inline src/assets/global.scss:32]: Do we still need this file?

File: src/components/layout/Button.vue [hunk 5]
```diff
@@ -0,0 +1,19 @@
+<template>
+  <button  :class="btnClasses ? btnClasses : 'inline-flex gap-2 items-center justify-center bg-peach w-full rounded-md border border-transparent py-2 px-4 text-base font-semibold text-black shadow-sm hover:bg-peach-2 focus:outline-none focus:ring-2 focus:ring-peach focus:ring-offset-2 ease-in-out duration-300'"
+    @click="onClick">
```
Comment (giteshk11) [inline src/components/layout/Button.vue:3]: It should emit an event instead of passing a function as prop.

Comment (giteshk11) [reply src/components/layout/Button.vue:3]: Also it should be able to handle icons on both left and right of the text, using slots

File: src/components/layout/Input.vue [hunk 6]
```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="space-y-1">
+  <label :class="labelClasses ? labelClasses : 'block text-sm font-normal text-grey-900 text-start'" :for="id">{{ labelText }}</label>
+  <input
+    :type="type"
+    :value="modelValue"
+    @change="onChange"
```
Comment (giteshk11) [inline src/components/layout/Input.vue:7]: Emit an event instead of expecting a function

**Review by giteshk11 — CHANGES_REQUESTED** [review]

File: package.json [hunk 7]
```diff
@@ -9,34 +9,26 @@
...
+    "@tailwindcss/typography": "^0.5.8",
     "@vitejs/plugin-vue": "^3.2.0",
-    "@vue/eslint-config-prettier": "^7.0.0",
     "@vue/test-utils": "^2.2.4",
     "autoprefixer": "^10.4.13",
-    "eslint": "^8.22.0",
-    "eslint-plugin-vue": "^9.3.0",
     "jsdom": "^20.0.3",
     "postcss": "^8.4.19",
-    "prettier": "^2.7.1",
```
Comment (giteshk11) [inline package.json:38]: why are these deleted?

**Review by giteshk11 — CHANGES_REQUESTED** [review]
All your files have merge issues, you need to resolve them

File: package.json (hunk 7 above)
Comment (kshekh) [reply package.json:38]: These are not required for Frontend and UI development. If your development team require the then you can install.

Comment (kshekh) [reply package.json:38]: What are the conflicts/issues you have in merging... Can you share?

Comment (kshekh) [reply package.json:38]: We are here to develop frontend and UI. So please expect us to UI and fronted related issues fixes.
Now, we should start 2nd milestone.

**Review by giteshk11 — APPROVED** [review]
